### PR TITLE
Add intercepting sidebar recommendations

### LINKED ISSUE #164: Feature: suggested Blue Check account not collected
blue checks suggested when visit an account profile isn't collected. This is very useful fro collecting multiple of them quickly

### PR DESCRIPTION
**Pull request by rougetimelord**
Adds checking the users that get recommended in the sidebar, closes #164.

So far everything works in my testing! :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "blue-blocker",
-	"version": "0.4.9",
+	"version": "0.4.10",
```
Comment (cooljeanius) [inline package.json:3]: why not go to 0.5.0 next?

Comment (rougetimelord) [reply package.json:3]: I guess I don't think it's a big enough change for a minor version, and semver lets you have non-single digit parts so why not?

Comment (cooljeanius) [reply package.json:3]: well I was wondering if maybe you might want to include some other changes as well?

Comment (cooljeanius) [reply package.json:3]: eh whatever

**Review by cooljeanius — APPROVED** [review]

**Pull request merged**